tensorflow/python/ops/array_ops.py

```diff
@@ -3069,7 +3069,8 @@ def sparse_placeholder(dtype, shape=None, name=None):
     print(sess.run(y, feed_dict={
       x: (indices, values, shape)}))  # Will succeed.
 
-    sp = tf.sparse.SparseTensor(indices=indices, values=values, dense_shape=shape)
+    sp = tf.sparse.SparseTensor(indices=indices, values=values,
+     dense_shape=shape)
```

Comment:
```suggestion
    sp = tf.sparse.SparseTensor(indices=indices, values=values,
                                                  dense_shape=shape)
```

Reply:
Please make sure you don't mix tabs and space either (I only inserted spaces above, but it seems there is a tab? The goal was to align `dense_shape` to the same column as `indices=`)

Reply:
Updated as suggested, thanks.